sonaranalyzer-dotnet/src/Tests/SonarAnalyzer.UnitTest/TestCases/MethodParameterUnused.Fixed.cs

```diff
@@ -53,6 +53,11 @@ public void Caller()
         {
         }
 
+        static async Task Main(string[] args) // Compliant - new main syntax

```

Comment:
Additional test cases:
 _Task<int>_ should be compliant
 _Task<string> should not

Reply:
Good catch too. I will add `Task<int>`.